includes/data-events/connectors/class-connector.php

```diff
@@ -51,7 +51,7 @@ public static function reader_registered( $timestamp, $data, $client_id ) {
 			'metadata' => $metadata,
 		];
 
-		static::put( $contact );
+		static::put( $contact, 'RAS Reader registration' );
```

Comment:
```suggestion
		static::put( $contact, sprintf( 'RAS Reader registration. User ID: %d. Data: %s.', $data['user_id'], wp_json_encode( $metadata ) ) );
```

Reply:
Not sure about adding the metadata here. The context should not be too long and not human readable. It's the "why". The metadata should be added in that extra field. Isn't it there already? I'll have a look